src/pages/computer-hardware/cpu/index.md

```diff
@@ -1,15 +1,20 @@
 ---
 title: Cpu
 ---
-## Cpu
+## CPU
 
-This is a stub. <a href='https://github.com/freecodecamp/guides/tree/master/src/pages/computer-hardware/cpu/index.md' target='_blank' rel='nofollow'>Help our community expand it</a>.
+CPU stands for Central Processing Unit -- it is the "brain" of the computer. 
 
-<a href='https://github.com/freecodecamp/guides/blob/master/README.md' target='_blank' rel='nofollow'>This quick style guide will help ensure your pull request gets accepted</a>.
+CPUs are integrated circuits, which are complex circuits embedded on a single chip. They include registers, which store single values, and input/output pins. These pins allow them to talk to the rest of the computer. Often times CPUs will also contain more than one core, which allows them to multitask. 
+
+CPU speeds are in gigahertz (GHz). For every gigahertz of speed, a CPU can do on billion operations in a second. If your CPU has a speed of 3 GHz, that means it performs three billion tasks every second. These tasks are very simple, such as "add two numbers" or "move this variable to this location." To see this in action, read about <a href='https://en.wikipedia.org/wiki/Assembly_language'>assembly language</a>.
+
+
+Almost all CPUs use a 64-bit architecture, which means they store information as 64 binary 'bits'. Older CPUs used 32-bit, 16-bit, and even 8-bit architectures. The largest number a 64-bit CPU can store is 18,446,744,073,709,552,000. 
 
 <!-- The article goes here, in GitHub-flavored Markdown. Feel free to add YouTube videos, images, and CodePen/JSBin embeds  -->
 
 #### More Information:
-<!-- Please add any articles you think might be helpful to read before writing the article -->
+https://en.wikipedia.org/wiki/Central_processing_unit
```

Comment:
Could you make this a link for me please

In markdown it would be `[this text is what I would click](https://the-url-of-the-awesome-resource.com)`